source/_integrations/zwave_me.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: Z-Wave.Me Z-Way
+description: Instructions on how to integrate Z-Wave with Home Assistant via Z-Wave.Me Z-Way.
+featured: true
+ha_category:
+  - Binary Sensor
+  - Climate
+  - Cover
+  - Fan
+  - Hub
+  - Light
+  - Lock
+  - Number
+  - Select
+  - Sensor
+  - Siren
+  - Switch
+ha_release: 2022.2
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@lawfulchaos, @Z-Wave-Me'
+ha_domain: zwave_me
+ha_platforms:
+  - binary_sensor
+  - climate
+  - cover
```

Comment:
```suggestion
```

Reply:
Looks like we forgot this platform. We will add it.